src/AppBundle/Enem/EnemFilter.php

```diff
@@ -19,9 +20,12 @@ class EnemFilter
         2009
```

Comment:
Shouldn't it be the same range in `EnemFilter` and `EnemEditionSelected`?

Reply:
Yes, they should. I'll fix.